### PR TITLE
Add `drvr_lic_type_id` and `veh_hnr_fl` columns

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/18931

## Testing

1. Start your local stack
2. Apply migrations and metadata
3. From your CRIS import directory,  run the import on whatever zips you have laying around (or use `--s3-download` to fetch one from the bucket).

```shell
$ docker run -it  --rm --env-file .env --network host -v $PWD:/app cris_import_cris_import ./cris_import.py --csv --skip-unzip
```

4. To make sure the trigger is working, inspect the data in your DB by setting the `created_at` date in this query to the current date:

```sql
-- compare cris drvr_lic_type_id to unified value
SELECT
    people_cris.drvr_lic_type_id AS cris_drvr_lic_type_id,
    people.drvr_lic_type_id AS unified_drvr_lic_type_id
FROM
    people_cris
    LEFT JOIN people ON people_cris.id = people.id
WHERE
    people_cris.created_at > '2024-09-10';

-- compare cris cris_veh_hnr_fl to unified value
SELECT
    units_cris.veh_hnr_fl AS cris_veh_hnr_fl,
    units.veh_hnr_fl AS unified_veh_hnr_fl
FROM
    units_cris
    LEFT JOIN units ON units_cris.id = units.id
WHERE
    units_cris.created_at > '2024-09-10';


```

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vzd/migrations/default/1725990797742_veh_hnr_fl/up.sql [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        unit_nbr,
+        updated_by,
+        veh_body_styl_id,
+        veh_damage_description1_id,
+        veh_damage_description2_id,
+        veh_damage_direction_of_force1_id,
+        veh_damage_direction_of_force2_id,
+        veh_damage_severity1_id,
+        veh_damage_severity2_id,
+        veh_hnr_fl,
```
Comment (johnclary) [inline atd-vzd/migrations/default/1725990797742_veh_hnr_fl/up.sql:47]: here's the new field

File: atd-vzd/migrations/default/1725990797742_veh_hnr_fl/up.sql [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        new.updated_by,
+        new.veh_body_styl_id,
+        new.veh_damage_description1_id,
+        new.veh_damage_description2_id,
+        new.veh_damage_direction_of_force1_id,
+        new.veh_damage_direction_of_force2_id,
+        new.veh_damage_severity1_id,
+        new.veh_damage_severity2_id,
+        new.veh_make_id,
+        new.veh_hnr_fl,
```
Comment (johnclary) [inline atd-vzd/migrations/default/1725990797742_veh_hnr_fl/up.sql:83]: and here's the new field

File: atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/up.sql [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        prsn_taken_by,
+        prsn_taken_to,
+        prsn_type_id,
+        unit_id,
+        updated_by
+    ) values (
+        new.created_by,
+        new.drvr_city_name,
+        new.drvr_drg_cat_1_id,
+        new.drvr_lic_type_id,
```
Comment (johnclary) [inline atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/up.sql:87]: and here's the new field

**Review by chiaberry — APPROVED** [review]
new columns, ✅

File: atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/up.sql [hunk 4]
```diff
@@ -0,0 +1,123 @@
+create table lookups.drvr_lic_type (
+    id integer primary key,
+    label text not null unique,
```
Comment (mddilley) [inline atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/up.sql:3]: i always forget that you can name constraints implicitly like this. Nice that it takes away the headache of naming yet another thing!

File: atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/down.sql
[the remark this answers is not in the document]
Comment (mddilley) [reply atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/down.sql:1]: looks great!! i read through the section about adding new columns, and it lines right up with what is going on in this PR. 💯

**Review by mddilley — APPROVED** [review]
I tested the migrations up and down, imported data, and saw values in the new columns. Very cool to see what capturing a new field from the extracts looks like! 🚀 🚢

**Comment by johnclary** [comment]
Thank you!!!

**Pull request merged**